core/sql/common/ComMisc.cpp

```diff
@@ -99,3 +99,17 @@ NABoolean ComIsTrafodionReservedSchemaName(
   return FALSE;
 }
 
+// schema names of pattern "_HV ... _" and "_HB_ ... _" are reserved to store
```

Comment:
Nit: Comment is not quite right... it's _HV_ and _HB_ (the trailing underscore after _HV is missing).